wiki/templates/wiki/dir.html

```diff
@@ -22,14 +22,14 @@
       {% trans "Add article" %}
     </a>
   </div>
-  <div class="pull-right">
-    {{ filter_form.query }}
+  <div class="pull-right input-prepend">
+    {{ filter_form.query.label_tag }} {{ filter_form.query }}
+    {% if filter_query %}
```

Comment:
Previously "(clear)" was appearing to the left of the input field when it displayed.

![image](https://cloud.githubusercontent.com/assets/484484/22742671/8a98acec-ede5-11e6-9539-89290353dca3.png)